src/jacksettings.py

```diff
@@ -404,9 +404,9 @@ def loadServerSettings(self, reset=False, forceReset=False):
             elif attribute == "alias":
                 self.ui.obj_server_alias.setChecked(bool(value))
             elif attribute == "client-timeout":
-                self.setComboBoxValue(self.ui.obj_server_client_timeout, str(value))
+                self.setComboBoxValue(self.ui.obj_server_client_timeout, str(int(value)))
             elif attribute == "clock-source":
-                value = str(value)
+                value = str(int(value))
```

Comment:
I was just working on this because I could not save jack settings. Thanks for that.

I think this line is a bit confusing. Based on this, the following checks for "c", "h", "s", will never succeed. I am not sure what type the jack dbus interface is supposed to return for clock-source.

What I did is use `str(value) == "c"` for the checks instead, and then in the else block make `int(value) == JACK_TIMER_*` checks.

Reply:
Thanks for the review, please tell me what you think of the updated code.

Reply:
That should work fine, also fixed the xruns text, nice :)

Reply:
Ok cool, do not hesitate if there is something missing or wrong ;)

Reply:
oh, about the xruns, there is also one more place that needs to be changed, at line 2337.

Reply:
Please open a new conversation with the quoted code so I can see better please.